uktrade/data-hub-frontend
# Add export history page

## Description of change

Adds the export history page. 

This is a hidden URL at the moment. There is a seperate PR to add the link to this page to the export page.

## Test instructions

You should see the export countries history using the collectionList component when going to /companies/:companyId/exports/history

There is only history from when we merged the history PR last week - so you may have to add and delete some export countries to see the feature in action. 

You also need the `export-countries-switch` and `interaction-add-countries` feature flags on. 

## Screenshots
### Before

![FireShot Capture 034 - Full export history - Exports - One List Subsidiary Ltd - Companies -_ - localhost](https://user-images.githubusercontent.com/42253716/74151016-1f2ee000-4c03-11ea-96df-dc8ed0687db8.png)

### After

![FireShot Capture 035 - Full export history - Exports - One List Subsidiary Ltd - Companies -_ - localhost](https://user-images.githubusercontent.com/42253716/74261302-f7b74080-4cf2-11ea-9011-e63f549f4fa4.png)

## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"
